ONSdigital/perkin
# Login once when putting a collection of files

See issue https://github.com/ONSdigital/perkin/issues/107

**Changes**

- Add login / logout methods to ftppublisher
- Use login/logout when transferring a collection of files at the start/end of transaction

**How to test**

- This can be tested within the docker-compose setup to determine if it is functionally the same, but ideally we'd want to test under preprod to confirm whether the login overhead has been reduced for each submission.

**Who can test**

Anyone but @iwootten
